pupil-spa/src/environments/environment.prod.ts

```diff
@@ -1,6 +1,13 @@
 export const environment = {
   apiURL: 'API_URL',
   production: true,
-  apiErrorDelay: 2000,
-  apiErrorMaxAttempts: 3
+  // Delay and max attempts for /api/completed-check api post method
+  checkStartAPIErrorDelay: 2000,
+  checkStartAPIErrorMaxAttempts: 3,
+  // Delay and max attempts for /api/check-started api post method
+  checkSubmissionApiErrorDelay: 30000,
```

Comment:
can you please specify the unit as Jon suggested (milliseconds etc)

Reply:
@GuyHarwood it's updated